content/it/autoscaling.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Autoscaling
+status: Completed
+category: Proprietà
+---
+
+_Autoscaling_ è la capacità di un sistema di [scalare](/scalabilità/) automaticamente, in genere, in termini di risorse di elaborazione. Con un sistema di scalabilità automatica, le risorse vengono aggiunte automaticamente quando necessario e possono essere ridimensionate per soddisfare le fluttuanti richieste degli utenti. Il processo di scalabilità automatica varia ed è configurabile per scalare in base a diverse metriche, come la memoria o il tempo di elaborazione. I servizi cloud gestiti sono in genere associati alla funzionalità di scalabilità automatica poiché sono disponibili più opzioni e implementazioni rispetto alla maggior parte delle distribuzioni locali (_on-premise_).
```

Comment:
Il verbo scalare in italiano non ha significato di 'to scale'. Io userei una circonlocuzione
```suggestion
_Autoscaling_ è la capacità di un sistema di essere [scalabile](/scalability/) in maniera automatica, generalmente inteso in termini di risorse di elaborazione. Con un sistema di scalabilità automatica, le risorse vengono aggiunte automaticamente quando necessario e possono essere ridimensionate per soddisfare le richieste fluttuanti degli utenti. Il processo di scalabilità automatica varia ed è configurabile per reagire a diverse metriche, come la memoria o il tempo di elaborazione. I servizi cloud gestiti sono in genere associati alla funzionalità di scalabilità automatica poiché sono disponibili più opzioni e implementazioni rispetto alla maggior parte delle distribuzioni _on-premise_ (locali).
```

Reply:
Come già altrove fatto presente, ormai "scalare" come calco di "to scale" è ormai valido 😎
Nonostante ciò, mi piace anche la parafrasi di @sistella 😝